frontend/simple/model/state.js

```diff
@@ -172,6 +172,13 @@ const getters = {
       if (!groupId) return 0
       return Object.keys(state[groupId].profiles).length
     }
+  },
+  proposalData (state, getters) {
+    return (proposalHash, groupId) => {
+      if (!groupId) groupId = state.currentGroupId
+      if (!groupId) return 0
```

Comment:
Why do we return zero? Shouldn't we return an empty object or `null`?

Reply:
Yes, ideally the type of data returned by these functions should be the same, and feel free to add it as a [Flowtype annotation](https://flow.org/en/docs/types/functions/) if you'd like (I personally recommend it, for even if the flowtype checker doesn't catch the issue, it provides great inline documentation on how the function is supposed to behave).

Reply:
you're totally right! this was some careless copypasting on my end. Flow is also a great idea 👍